site/src/components/WorkspaceScheduleButton/EditHours.tsx

```diff
@@ -19,13 +19,13 @@ export const EditHours = ({
   const styles = useStyles()
 
   return (
-    <form onSubmit={() => handleSubmit(hours)}>
+    <form onSubmit={() => handleSubmit(Number.isNaN(hours) ? 0 : hours)}>
```

Comment:
when is it going to be `NaN`?

Reply:
When you delete the number from the input

Reply:
Maybe we could add a comment explaining this? So we know why we are checking it. Up to you.

Reply:
Good idea!